tensorflow/python/keras/saving/hdf5_format_test.py

```diff
@@ -730,6 +730,48 @@ def test_model_saving_to_pre_created_h5py_file(self):
       os.close(fd)
       os.remove(fname)
 
+  def test_model_saving_to_new_dir_path(self):
+    saved_model_dir = self._save_model_dir()
+    saved_model_dir = os.path.join(saved_model_dir, 'newdir')
```

Comment:
os.path.join can take multiple paths. You can just join all the parts you need in one line.

Reply:
Thanks for the comments!